src/pages/MonthlyGood/MonthlyGoodSetupPage.vue

```diff
@@ -92,9 +92,13 @@
 											v-model="donationCheckbox"
 											@change="donationCheckboxChange()"
 										/>
-										<span
-											class="additional-left-pad-spans display-inline-block"
-										>{{ donationTagLine }}</span>
+										<span class="additional-left-pad-spans display-inline-block">
```

Comment:
Because you have a block level element (a div) inside of this inline element, I'd recommend changing this span to a div.